traffic_updater/go/wayid2nodeid_extractor/wayid2nodeids_extractor.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"math/rand"
+	"os"
+	"runtime"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/qedus/osmpbf"
+)
+
+const telenavNavigableWaySuffix string = "100"
+
+func generateWayid2nodeidsMapping(input, output string) {
+	infile, err := os.Open(input)
+	defer infile.Close()
+	if err != nil {
+		log.Fatal(err)
+		fmt.Printf("Open pbf file of %v failed.\n", input)
+		return
+	}
+	fmt.Printf("Open pbf file of %s succeed.\n", input)
+
+	outfile, err := os.OpenFile(output, os.O_RDWR|os.O_CREATE, 0755)
+	defer outfile.Close()
+	defer outfile.Sync()
+	if err != nil {
+		log.Fatal(err)
+		fmt.Printf("Open output file of %s failed.\n", output)
+		return
+	}
+	fmt.Printf("Open output file of %s succeed.\n", output)
+
+	wayid2nodeids(infile, outfile)
+}
+
+func wayid2nodeids(infile io.Reader, outfile io.Writer) {
+	// Init extractor
+	extractor := osmpbf.NewDecoder(infile)
+	extractor.SetBufferSize(osmpbf.MaxBlobSize)
+	err := extractor.Start(runtime.GOMAXPROCS(-1))
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+
+	// Init loader
+	loader := bufio.NewWriter(outfile)
+	defer loader.Flush()
+
+	var wc, nc uint32
+	for {
+		if v, err := extractor.Decode(); err == io.EOF {
+			break
+		} else if err != nil {
+			log.Fatal(err)
+		} else {
+			switch v := v.(type) {
+			case *osmpbf.Node:
+			case *osmpbf.Way:
+				wayid := strconv.FormatUint((uint64)(v.ID), 10)
+				if !isNavigableWay(wayid) {
+					continue
+				}
+				wayid = trimNavigableWaySuffix(wayid)
+
+				// Transform
+				//str := convertWayObj2IdMappingString(v, wayid)
+				str := convertWayObj2MockSpeed(v, wayid)
```

Comment:
Here should be use `convertWayObj2IdMappingString` instead of `convertWayObj2MockSpeed`, right?